sip/sip-005-blocks-and-transactions.md

```diff
@@ -889,42 +903,32 @@ the Stacks peer tracks the following sets of key/value pairs:
 * the mapping between account addresses and their nonces and asset maps
 * the mapping between fully-qualified smart contract names and a bundle of
   metadata about them (described below).
-* the mapping between fully-qualified data variable names and their values
-* the mapping between fully-qualified data map keys and their values
+* the mapping between fully-qualified smart contract data keys and their 
+  associated values. 
 
 The first set of key/value pairs is the **account state**.  The Stacks peer
 calculates an index over all accounts in each fork as they are created.
 
-The second set of key/value pairs is the **smart contract state**.  It maps the
+The second set of key/value pairs is the **smart contract context state**.  It maps the
 _fully-qualified name_ of the smart contract to a bundle of metadata,
 which contains:
    * the transaction ID that created the smart contract (which can be used to
      derive the contract account address and to query its code),
-   * the list of data variable names defined by this contract
-   * the list of data maps defined by this contract
-   * the list of assets defined by this contract
-   * the list of tokens defined by this contract
+   * all functions and constants defined by this contract
```

Comment:
Are we still doing this?  I thought we were only going to commit to the TXID, the persistent state values (variables, data maps, and assets), and (soon) the contract nonce?

Reply:
Ah, yeah, I thought I had removed this, but hadn't. It's been updated now.